### PR TITLE
Change the description of account recovery in My Account

### PR DESCRIPTION
**Pull request by dasuni-30**
### Related Issues
- None

### Checklist
- [ ] e2e cypress tests locally verified.
- [X] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- None

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/myaccount/src/components/account-recovery/account-recovery.tsx [hunk 1]
```diff
@@ -161,7 +162,11 @@ export const AccountRecoveryComponent: FunctionComponent<AccountRecoveryProps> =
             {
                 <SettingsSection
                     data-testid={ `${testId}-settings-section` }
-                    description={ t("myAccount:sections.accountRecovery.description") }
+                    description={
+                        accountRecoveryConfig.sections.accountRecovery.description == undefined ?
+                        t("myAccount:sections.accountRecovery.description") 
+                        : accountRecoveryConfig.sections.accountRecovery.description
```
Comment (thivi) [inline apps/myaccount/src/components/account-recovery/account-recovery.tsx:168]:
```suggestion
                        accountRecoveryConfig.sections.accountRecovery.description == undefined 
                        ? t("myAccount:sections.accountRecovery.description") 
                        : accountRecoveryConfig.sections.accountRecovery.description
```

File: apps/myaccount/src/extensions/configs/account-recovery.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
+ *
+ * This software is the property of WSO2 Inc. and its suppliers, if any.
+ * Dissemination of any information or reproduction of any material contained
+ * herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+ * You may not alter or remove any copyright or other notice from copies of this content."
+ */
+
```
Comment (thivi) [inline apps/myaccount/src/extensions/configs/account-recovery.ts:9]: The license header should be different.

File: apps/myaccount/src/extensions/configs/models/account-recovery.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
+ *
+ * This software is the property of WSO2 Inc. and its suppliers, if any.
+ * Dissemination of any information or reproduction of any material contained
+ * herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+ * You may not alter or remove any copyright or other notice from copies of this content."
+ */
```
Comment (thivi) [inline apps/myaccount/src/extensions/configs/models/account-recovery.ts:8]: The license header should be different.

**Review by thivi — CHANGES_REQUESTED** [review]

File: apps/myaccount/src/components/account-recovery/account-recovery.tsx (hunk 1 above)
Comment (dasuni-30) [reply apps/myaccount/src/components/account-recovery/account-recovery.tsx]: Addressed the change.

File: apps/myaccount/src/extensions/configs/account-recovery.ts (hunk 2 above)
Comment (dasuni-30) [reply apps/myaccount/src/extensions/configs/account-recovery.ts]: Addressed the change.

File: apps/myaccount/src/extensions/configs/models/account-recovery.ts (hunk 3 above)
Comment (dasuni-30) [reply apps/myaccount/src/extensions/configs/models/account-recovery.ts]: Addressed the change.

**Review by thivi — APPROVED** [review]

File: apps/myaccount/src/extensions/configs/account-recovery.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+ * under the License.
+ */
+
+import { AccountRecoveryConfig } from "./models";
+
+export const accountRecoveryConfig: AccountRecoveryConfig = {
+    sections: {
+        accountRecovery: {
+            description:
+                undefined
```
Comment (yasinmiran) [inline apps/myaccount/src/extensions/configs/account-recovery.ts:25]: > Inline the value

File: apps/myaccount/src/extensions/configs/models/index.ts [hunk 5]
```diff
@@ -17,3 +17,4 @@
  */
 
 export * from "./common";
+export * from "./account-recovery";
```
Comment (yasinmiran) [inline apps/myaccount/src/extensions/configs/models/index.ts:20]: > Add newline

Comment (dasuni-30) [reply apps/myaccount/src/extensions/configs/models/index.ts:20]: A new line is already added here. It is not shown in Github.

File: apps/myaccount/src/extensions/configs/account-recovery.ts (hunk 4 above)
Comment (dasuni-30) [reply apps/myaccount/src/extensions/configs/account-recovery.ts:25]: Fixed by https://github.com/wso2/identity-apps/pull/3061

**Pull request merged**